import_export/resources.py

```diff
@@ -546,8 +546,8 @@ def skip_row(self, instance, original):
         """
         Returns ``True`` if ``row`` importing should be skipped.
 
-        Default implementation returns ``False`` unless skip_unchanged == True,
-        or skip_diff == True.
+        Default implementation returns ``False`` unless skip_unchanged == True
+        and skip_diff == False.
```

Comment:
I see what you mean here. The thing is that the default value for `skip_unchanged` is `False`, and the default value for `skip_diff` is `False`. So changing any of those variable values will skip the default implementation and check for changes to decide if the row has to be skipped. I'd say you are right here too, but we should emphasise default values, something like this:

    Default implementation returns ``False`` unless skip_unchanged == True (defaults to False)
    and skip_diff == False (default).

Reply:
Thanks for your feedback. I agree, howevery I think it is quite a lot of information for one sentence. How about:
```
Default implementation returns ``False`` unless skip_unchanged == True 
and skip_diff == False. 

Left unspecified, skip_diff``  and skip_unchanged`` both default to ``False``, 
and rows are never skipped. 
```

Reply:
Much better!

Reply:
Changes are pushed!